src/test/scala/cromwell/engine/WorkflowManagerActorSpec.scala

```diff
@@ -115,4 +115,17 @@ class WorkflowManagerActorSpec extends CromwellTestkitSpec("WorkflowManagerActor
       }
     }
   }
+
+  it should "error when running a workflowless WDL" {
+    DataAccess.withDataAccess { dataAccess =>
+      implicit val workflowManagerActor = TestActorRef(WorkflowManagerActor.props(dataAccess), self, "Test a workflowless submission")
+      Try(messageAndWait[WorkflowId](SubmitWorkflow(HelloWorldWithoutWorkflow.wdlSource(),
+        HelloWorldWithoutWorkflow.wdlJson, HelloWorldWithoutWorkflow.rawInputs))) match {
+        case Success(_) => fail("Expected submission to fail due to no runnable workflows")
+        case Failure(e) => e.getMessage shouldBe "Namespace does not have a local workflow to run"
+      }
+
+      0 // FIXME: Why is this expecting an int as return type?
```

Comment:
You can get rid of that trailing 0 if you move this inside the existing `should` and change the bracketing code to:

```
    "error when running a workflowless WDL" in {
```